roles/online_docs/templates/mkdocs/docs/apptainer.md

```diff
@@ -0,0 +1,595 @@
+
+4. Execute custom command inside the container
+
+    `$ apptainer exec alpine_latest.sif cat /etc/os-release`
+
+5. Execute a command directly from the Docker's repository container
+
+    `$ apptainer exec docker://busybox:latest busybox | head -n1`
+
+6. Build sandbox directory directly from the Docker image
+
+    `$ apptainer build --sandbox alpine_sandbox docker://busybox:latest`
+
+7. Shell into the sandbox, pretend to be a root user and permanently save all changes
+
+    `$ apptainer shell --fakeroot --writable alpine_sandbox`
+
+8. Convert sandbox to static image
+
+    `$ apptainer build alpine.sif alpine_sandbox`
+
+## Building Apptainer images
+
+In order to use apptainer containers, the image first needs to be built. Images can be built on top of another images (they can be local images or from the Docker repository), which can be done without elevated permissions. On the other hand, if an image is being built from scratch with f.e. `yum` installer, the elevated permissions are needed on the system to execute `sudo apptainer build ...`. 
+
+Normally, regular users don't have elevated permissions on multi-user systems, which is also true on {{ slurm_cluster_name | capitalize }}. Section `Building without elevated permissions` describes how to build images without elevated permissions.
+
+### Building without elevated permissions
+
+At the time of writing this documentation (late 2022) the Apptainer documentation exaplains the option of using a `--fakeroot` parameter as a way to bypass some of these limitations. The latest version `1.1.4` has still some know issues with this option and will hopefully be soon resolved.
+
+Fortunately, there are at least two alternatives, as users can either:
+- simply build an `.sif` image file on a system where they have elevated (sudo) permissions, and then copy that `.sif` file to the cluster, or
+- build an image on the cluster itself, but doing via the additional step of using the sandbox option and then convert the sandbox into the `.sif` file. To use this option, read sections `Sandbox` and `Building image as regular user`.
+
+## Sandbox
+
+The sandbox option builds a container image inside a directory.
+
+The sandbox option allows users to bypass some of the limitations, as the sandbox container allows user
+
+* to run fakeroot option and become root user inside container,
+* to change any files inside the container, and those changes *can be* (if used `--writable`) permanently saved,
+* can convert sandbox back to `.sif` file.
+
+The drawbacks on the other hand are
+
+* the changes inside the sandbox (in contrast to the build from `.def` files) are not `recorded` in the image, and therefore,
+* the `.sif` file is hard to maintain in the long run,
+* running sandbox often brings lower performance: due to the converting of the image, and (in case of shared remote file system use) the access of individual files inside the sandbox is slower than the use of .sif format.
```

Comment:
I do not understand this. Does this only apply to running a sandbox directory or does this also apply when the sandbox is converted back to a `*.sif`?